MirrorProvider/Scripts/Linux/MirrorProvider_Mount.sh

```diff
@@ -8,4 +8,6 @@ fi
 SCRIPTDIR=$(dirname ${BASH_SOURCE[0]})
 BUILDDIR=$SCRIPTDIR/../../../../BuildOutput/MirrorProvider.Linux/bin/$CONFIGURATION/x64/netcoreapp2.1
 
-dotnet $BUILDDIR/MirrorProvider.Linux.dll mount ~/TestRoot
+TEST_ROOT="${TEST_ROOT:=~/TestRoot}"
```

Comment:
TIL about this `:=` trick. Thanks!

Reply:
This is new to me as well 😄 

After reading up on `:=`, is there a reason to prefer it to `:-`? Or would they be equivalent with how we're using them here?

Reply:
I suspect we could use `:-` here since we're assigning the output of the quote-delimited string back to `TEST_ROOT`, so `:=` actually performs an extra unused assignment *within* the `${...}` block.  Quick demo:
```
DASH="${TEST:-foo}"
echo $DASH $TEST
EQUAL="${TEST:=bar}"
echo $EQUAL $TEST
```
Output:
```
foo
bar bar
```

The `man bash` page is truly a treatise and there's always something in there I've never seen before, even though I've read it hundreds of times.  This week's learning experience for me was `set -u`, among several others.

Reply:
I'm absolutely no bash wizard (and pulled this from some random internal script which is configured with env vars). Happy to defer to the experts!